Source/main.cpp

```diff
@@ -12,12 +12,14 @@ using namespace amrex;
 
 int main(int argc, char* argv[])
 {
+#if defined AMREX_USE_MPI
 #if defined(_OPENMP) && defined(WARPX_USE_PSATD)
     int provided;
     MPI_Init_thread(&argc, &argv, MPI_THREAD_FUNNELED, &provided);
     assert(provided >= MPI_THREAD_FUNNELED);
 #else
     MPI_Init(&argc, &argv);
+#endif
 #endif
```

Comment:
Do we want to start using a convention for indenting pre-processor directives, e.g.,
```c++
#if defined AMREX_USE_MPI
#   if defined(_OPENMP) && defined(WARPX_USE_PSATD)

    int provided;
    MPI_Init_thread(&argc, &argv, MPI_THREAD_FUNNELED, &provided);
    assert(provided >= MPI_THREAD_FUNNELED);

#   else

    MPI_Init(&argc, &argv);

#  endif
#endif
```
as @ax3l did in PR https://github.com/ECP-WarpX/WarpX/pull/331 or should we do it for `#include` only?